compiler/ir/backend.js/src/org/jetbrains/kotlin/ir/backend/js/JsCommonBackendContext.kt

```diff
@@ -134,6 +134,8 @@ interface JsCommonInlineClassesUtils : InlineClassesUtils {
      */
     fun getInlinedClass(type: IrType): IrClass?
 
+    fun getInlinedClassOrThrow(type: IrType): IrClass = getInlinedClass(type) ?: error("Unable to get inlined class for type $type")
```

Comment:
```suggestion
    fun getInlinedClassOrThrow(type: IrType): IrClass = getInlinedClass(type) ?: error("Unable to get inlined class for type ${type.render()}")
```
You can use `render` here, as we do in many other places in the project, and avoid changes in `IrTypeBase`

Reply:
Here - where? As I see, I have used `type.render()`

Reply:
```
[...] ?: error("Unable to get inlined class for type $type")
                                                     ^^^^^
```
Here you're using `IrType.toString`. I propose to use `render` instead.